recipes/default.rb

```diff
@@ -359,16 +359,16 @@
   action :run
   command 'php scripts/composer_wrapper.php install --no-dev'
   cwd librenms_homedir
-  user 'root'
-  group 'root'
+  user librenms_username
```

Comment:
How do you guaranty this user and group exist before the chef run?

Reply:
They are created earlier in the same Cookbook.